### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_Eclipse Marketplace_Keyboard: As soon as the focus lands the list item of "All Markets" and "All Categories" drop-downs, it is getting selected without user action

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3362]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!
 
GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS3.2.1; #Keyboard; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200

**OS Details:** 
OS: Windows 10 (Version 1903, Build: 18362.175)
MAS Violated: MAS 3.2.1 

**Repro Steps: ​**
1) Open Eclipse application
2) Tab till Help and select "Eclipse Marketplace" from the list
3) Search/Find for 'Azure Toolkit for Eclipse' and press enter
4) Tab till "All Markets" drop-down (and "All Categories" drop-down) and expand it.
5) Using down arrow, navigate to the items present in the drop-down
6) Check if the list item is getting selected automatically without user action

**Actual Result: ​**
As soon as the focus lands the list item of "All Markets" and "All Categories" drop-downs, it is getting selected without user action 

**Expected Result: ​**
The list item should be selected only when user presses enter key

**User Impact: ​**
Users will find it difficult to access the page if the list item gets selected without any user action
 
**MAS Reference:**
[MAS 3.2.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2})

[Bug_3362_Eclipse_Marketplace_on Focus_Dropdowns.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451808/Bug_3362_Eclipse_Marketplace_on.Focus_Dropdowns.pptx)

**Comment by Flanker32** [comment]
Hi @soumyatippapuram, The eclipse marketplace is controlled by eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here is a gif of search AWS in Eclipse marketplace, we could see the same behavior as this issue.


![3362](https://user-images.githubusercontent.com/12445236/64938731-71a4e400-d891-11e9-8d3a-979169e19fa8.gif)

**Comment by Flanker32** [comment]
- Importance of experience: Little, it's not a common behavior to change the market and categories
- Quality of experience: Medium, users may want to filer the result by markets or categories and it could be difficult for keyboard only users to do this job.
- Alternate solutions: Users could visit Eclipse market place in browser and the links could be accessible with tab
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@soumyatippapuram This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us verify this issue, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to Eclipse (https://bugs.eclipse.org/bugs/show_bug.cgi?id=551114)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749763
